index.d.ts

```diff
@@ -0,0 +1,149 @@
+export class js {
+
+  /**
+  * Sets the collision grid that EasyStar uses.
+  * 
+  * @param {Array|Number} tiles An array of numbers that represent 
+  * which tiles in your grid should be considered
+  * acceptable, or "walkable".
+  */
+  setAcceptableTiles(tiles: number[] | number): void
+  /**
+    * Enables sync mode for this EasyStar instance..
+    * if you're into that sort of thing.
+    */
+  enableSync(): void
+  /**
+  * Disables sync mode for this EasyStar instance.
+  */
+  disableSync(): void
+  /**
+   * Enable diagonal pathfinding.
+   */
+  enableDiagonals(): void
+
+  /**
+   * Disable diagonal pathfinding.
+   */
+  disableDiagonals(): void
+  
+  /**
+   * Sets the collision grid that EasyStar uses.
+   * 
+   * @param {Array} grid The collision grid that this EasyStar instance will read from. 
```

Comment:
Does TypeScript use these comments, or does it pull the comments from the `.js` itself ? I hate to have to duplicate documentation.